test/pluginsTests.ts

```diff
@@ -52,15 +52,4 @@ describe("Plugins", () => {
         expect(page.contents).with.lengthOf(3);
         expect((page.contents[2] as any).tag).to.equal("othertag");
     });
-
```

Comment:
Does KSS require actual files like the typescript plugin? Cuz if so then don't bother